### PR TITLE
Added short doc about css variables.

### PR DESCRIPTION
**Pull request by kchra**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [ ] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the [code standards](../CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

File: applications/common/frontend/README.md [hunk 1]
```diff
@@ -20,3 +20,17 @@ Front-End files are deployed as a part of `core` package.
 
 1. Build this module
 2. Build with local deployment `core` module
+
+
+## CSS Variables:
+
+CSS Variables are generated automaticly in 'global-variables.scss'. These variables can be `used` or `redefined` in your project.
```
Comment (Mikilll94) [inline applications/common/frontend/README.md]: typo in `automaticly`

File: applications/common/frontend/README.md [hunk 2]
```diff
@@ -20,3 +20,17 @@ Front-End files are deployed as a part of `core` package.
...
+
+Places where the data necessary to generate css variables are located:
+1. `/atomic-design-system/00-token/` - Contain folders named and arranged according to the purpose of css attributes. 
+2. `/bulma-overwrite/_variables.scss` - Contain defined spacing names and values.
+
+Important files:
+1. `global-variables.scss` - File responsible for generating css variables base on sass maps imported from '/atomic-design-system/00-token/*' and '/bulma-overwrite/_variables.scss'.
+
+IMPORTANT: The names of maps defined in the '/atomic-design-system/00-token/' and '/bulma-overwrite/_variables.scss' can't be changed because the rest of 'atomic-design-system' uses css variables from 'global-varibles.scss'.
```
Comment (Mikilll94) [inline applications/common/frontend/README.md]: typo in `global-varibles`

**Review by Mikilll94 — CHANGES_REQUESTED** [review]
I would write this in a different way. Here is my proposition.

Kyanite library defines a set of CSS variables. They can be used and overwritten in project using Kyanite. We can divide them into the following categories:

- colors

```css
background-color: var(--color-primary--200);
```

- font-size

```css
font-size: var(--font-size-12);
```

...

This is just my proposition. You can ignore it if you don't like it.

**Comment by paweljozwicki** [comment]
What do you think about adding information about how to overwrite those variables?

File: applications/common/frontend/README.md (hunk 1 above)
Comment (kchra) [reply applications/common/frontend/README.md]: fixed

File: applications/common/frontend/README.md (hunk 2 above)
Comment (kchra) [reply applications/common/frontend/README.md]: fixed

**Review by Mikilll94 — APPROVED** [review]

**Pull request merged**